### DISCUSSION TITLE
What is the 'View - Show Tab Bar' use for in macOS? Can I create tab window like broswer?

### DISCUSSION THREAD

**Discussion by chshouyu** [tauri-apps/tauri#4585] [Q&A] [answered]
<img width="281" alt="image" src="https://user-images.githubusercontent.com/1304342/177152017-3a973cc3-23fa-4c8b-930a-a41055da44d2.png">

I have searched the keyword 'tab' in docs and github, and found nothing. I want create tab window like browser, but don't know how to do it.

thanks.

**Comment by simonhyll** [comment, marked as answer]
The tabs that the browser uses isn't created in the Window itself. The closest you get is using an undecorated window and creating your own tab bar within the WebView. How you do that is up to whatever frontend framework you've chosen to decide
